### ISSUE TITLE
Conformance Challenges Goals Edit

### ISSUE THREAD

**Issue by lauracarlson** [w3c/wcag#1093]
The following statement under the [Goals](https://w3c.github.io/wcag/conformance-challenges/#goal) section seems like a declaration of truth that speaks for the full Working Group:

> We believe that a better understanding of the situations in which the WCAG 2.x conformance model may be difficult or impossible to apply, and the places where accessibility conformance verification scales poorly if at all, will lead to more effective conformance models and testing approaches in the future.

Suggest deleting "We believe" and changing "scales poorly if at all, will lead" to "present difficulties in scaling, could possibility lead".

So it would read:

> A better understanding of the situations in which the WCAG 2.x conformance model may be difficult or impossible to apply, and the places where accessibility conformance verification may present difficulties in scaling, could possibility lead to more effective conformance models and testing approaches in the future.

**Comment by peterkorn** [comment]
Thank you Laura, excellent suggestion.  Only change I would make to your proposed text is to swap "may" for "could possibility".  Thus it would then read:

A better understanding of the situations in which the WCAG 2.x conformance model may be difficult or impossible to apply, and the places where accessibility conformance verification may present difficulties in scaling, may lead to more effective conformance models and testing approaches in the future.

**Comment by lauracarlson** [comment]
Hi Peter,

That would be fine with me.

Thank you,

Laura

**Comment by guyhickling** [comment]
misspelling, should be "could possibly"

**Comment by sajkaj** [comment]
Text is now in [Editor's Draft](https://w3c.github.io/wcag/conformance-challenges/).

**Comment by lauracarlson** [comment]
Thank you, Janina.

On 4/7/20, sajkaj <notifications@github.com> wrote:
> Closed #1093.
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/w3c/wcag/issues/1093#event-3208935063


-- 
Laura L. Carlson
